R/unzipDwd.R

```diff
@@ -0,0 +1,102 @@
+################################################################################
+#' Extract files from a zip archive downloaded
+#'
+#' This function unzip dataset previously downloaded from url. Prior to unzip, the function can check if unzip
+#' was previously performed (unzip file exists locally).
+#'
+#' @param zipfile A character vector containing path to a zipfile.
+#' @param dir A character string representing the output directory path.
+#' @param env The environment to search for objects.
+#' @param checkhash A logical argument. If TRUE, check if unzip file is found locally and cross-check its checksum value
+#'        with value logged in dbHash from previous unzip event. When checksums match, a message raises indicating the file
+#'        is already properly unzip. When checksums doesn't match or unzip file doesn't exist locally, unzip occurs and
+#'        checksum compiles. If FALSE, unzip occurs even if the files exist locally. Default is FALSE.
+#' @param dbHash A character string. The path to the database file where checksum values are logged. If the named
+#'        database does not yet exist, one is created. Default is "dbHash.sqlite".
+#' @param quick A logical argument. If TRUE, checksum is compiled using the combination of the filename and its size.
+#'        If FALSE, cheksum is compiled using the object. Default is FALSE.
+#'
+#' @importFrom utils unzip
+#'
+#' @export
+#' @docType methods
+#'
+#' @author Melina Houle
+#' @examples
+#' datasetName ="AHCCD_daily"
+#' dfile = c("Homog_daily_mean_temp_v2015.zip")
+#' algo = "xxhash64"
+#' outdir <- outdir<-"C:/temp"
```

Comment:
All examples should be OS-independent (and machine-independent). Use `tempdir()` to get a tmp directory path.